snow/networking/throttler/ewma.go

```diff
@@ -15,23 +15,28 @@ import (
 )
 
 const (
-	defaultDecayFactor         float64 = 2
+	// Larger --> Consider past less for throttling
+	defaultDecayFactor         float64 = 3 / 2
```

Comment:
I believe that 3/2 in this case will evaluate to 1 before being cast to float64.

Reply:
good call